index.ts

```diff
@@ -1,9 +1,11 @@
-const infoLogger = require('./src/infoLogger');
-const warnLogger = require('./src/warnLogger');
-const errorLogger = require('./src/errorLogger');
-require('dotenv').config();
+import infoLogger from './src/infoLogger'
+import warnLogger from './src/warnLogger';
+import errorLogger from './src/errorLogger';
+import * as dotenv from 'dotenv'
+import { ErrorTypes } from './src/types/errorTypes';
+dotenv.config()
 
-function logger(context, message) {
+const logger = (context: string, message: string): void => {
```

Comment:
Criar um type com valores definidos na pasta de types, e usa aqui, algo como:

`export type ContextValues = 'info' | 'warn' | 'error'`

Assim no código em que a lib for instalada apareçerão as opções e strings inválidas não serão aceitas pelo Typescript.